app/Global.scala

```diff
@@ -73,65 +76,89 @@ object Global extends GlobalSettings with LazyLogging{
  * Initial set of data to be imported
  * in the sample application.
  */
-object InitialData extends GlobalDBAccess with LazyLogging {
-
-  val mpi = Team("Connectomics department", None, RoleService.roles)
-
-  def insert(conf: Configuration) = {
-    insertDefaultUser(conf)
-    insertTeams()
-    insertTasks()
-    if (conf.getBoolean("datastore.enabled").getOrElse(true)) {
-      insertLocalDataStore(conf)
-    }
-  }
-
-  def insertDefaultUser(conf: Configuration) = {
+object InitialData extends GlobalDBAccess with FoxImplicits with LazyLogging {
+
+  val defaultUserEmail = Play.configuration.getString("application.authentication.defaultUser.email").get
+  val defaultUserPassword = Play.configuration.getString("application.authentication.defaultUser.password").get
+  val rootTeamName = "Connectomics department"
+
+  def insert =
+    for {
+      _ <- insertDefaultUser
+      _ <- insertRootTeam
+      _ <- giveDeafultUserTeam
+      _ <- insertTaskType
+      _ <- insertProject
+      _ <- if (Play.configuration.getBoolean("datastore.enabled").getOrElse(true)) insertLocalDataStore else Fox.successful(())
```

Comment:
Maybe move this check to the `insertLocalDataStore` function itself too keep things clean here?